app/controllers/routeDetails/locationOfGoods/AdditionalIdentifierController.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.routeDetails.locationOfGoods
+
+import controllers.actions._
+import controllers.{NavigatorOps, SettableOps, SettableOpsRunner}
+import forms.AdditionalIdentifierFormProvider
+import models.{LocalReferenceNumber, Mode}
+import navigation.Navigator
+import navigation.annotations.PreTaskListDetails
+import pages.routeDetails.locationOfGoods.AdditionalIdentifierPage
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.SessionRepository
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.routeDetails.locationOfGoods.AdditionalIdentifierView
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class AdditionalIdentifierController @Inject() (
+  override val messagesApi: MessagesApi,
+  implicit val sessionRepository: SessionRepository,
+  @PreTaskListDetails implicit val navigator: Navigator,
```

Comment:
Navigator for this section is available. See other location of goods controllers for reference. Spec will need updating as well to bind the appropriate navigator